packages/schema/src/json-schema.ts

```diff
@@ -122,6 +122,8 @@ export const queryProperty = <T extends JSONSchema>(def: T) => {
   } as const
 }
 
+export const SUPPORTED_TYPES = ['string', 'number', 'integer', 'boolean', 'null']
```

Comment:
For MongoDB we might need `array` in the supported types. MongoDB can handle a query like `{ userIds: { $in: [1, 2, 3] } }`.

Reply:
That should already be the case. `$in` and `$nin` are arrays of the primitive type.